src/lib/fy-dump.c

```diff
@@ -292,9 +292,8 @@ void fyp_debug_dump_input(struct fy_parser *fyp, const struct fy_input_cfg *fyic
 				fyic->file.filename);
 		break;
 	case fyit_stream:
-		fyp_scan_debug(fyp, "%s: stream=\"%s\" fileno=%d chunk=%zu\n", banner,
-				fyic->stream.name, fileno(fyic->stream.fp),
-				fyic->stream.chunk);
+		fyp_scan_debug(fyp, "%s: stream=\"%s\" fileno=%d\n", banner,
```

Comment:
@pantoniou note the chunk is not here

Reply:
Ugh, yeah, you're right; serves me right for commenting on a phone...